June 2017 Inventory

mlenover opened the pull request:
Completed ARMPilot component inventory

mlenover committed:
June 2017 Inventory

Completed ARMPilot component inventory

scholvat reviewed (changes requested):
Couple of questions...
Commit name: Its called ZeroPilot, not ARMpilot :P
RESS-003: I'm pretty sure we don't have any of these... confirm that the ones we have are 1% (they are probably 5%)
RESS-006: We use them on the current picpilot so we should have a large quantity of these.
CAPP-010: We should have a good quantity of these. Make sure you look for either 10nF or 10000pF capacitors
RESS-001: There should be a clear plastic case on the shelf with the other containers with the connectors that contains a lot of SMD resistors. I'm pretty sure I saw some smd 0 ohm resistors in there, have you checked there?

The rest of it looks fine, just double check those components please.

uwuarg committed:
ZeroPilot Component Inventory Update

Re-inventoried parts to reflect identical components with different
Digikey part numbers. Also compiled list of similar components at end of
inventory.

scholvat reviewed (changes requested):
The actual data looks good, there is just some formatting needs to be done. If you look at the online csv viewer on github, it should be able to display the information properly. You have a few extra commas in line 18 I believe, and there are some weird characters in the field "Almost Components".

Can you rebase to master? Then I won't have to resolve any conflicts when it is merged in.

Also just to verify, you did actually find 1% 100KOhm resistors (RESS-003)? I didn't think we had any but serge might have ordered some in.

I know we have some bags of LEDs (DIOD-002 and DIOD-003) around the bay but I'm pretty sure we are running out of those anyways so we can order them regardless.

uwuarg committed:
Minor Inventory Changes

Confirmed we did not have any 100k 1% resistors. Removed apostrophe
characters so csv is formatted properly.

uwuarg committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Michael's-Inventory

# Conflicts:
#	component_library.csv

uwuarg committed:
Resolved conflicts with master

uwuarg committed:
Removed comma in row 18
